### ISSUE TITLE
Number of pages reports as 1 in 0.13.4

### ISSUE THREAD

**Issue by jakirkham** [blink1073/tifffile#34]
Recently ran into the following issue with `tifffile` version `0.13.4` where the code below will print `1` now instead of `150` like it should. This behavior differs from `tifffile` version `0.13.0` where the number of pages are reported correctly. Example file zipped and included below. Please let me know if there is anything else you need.

```python
import tifffile

t = tifffile.TiffFile("data.tif")
print(len(t.series[0].pages))
```

[data.tif.zip](https://github.com/blink1073/tifffile/files/1643997/data.tif.zip)

**Comment by blink1073** [comment]
Hi @jakirkham, the version in 0.13.0 was an older version of the source `tifffile.py`, accidentally published as a new version of this package.

**Comment by blink1073** [comment]
cc @cgohlke

**Comment by jakirkham** [comment]
Ah ok. Thanks for the info. So I guess it would be no surprise that the issue does not appear for me in `0.12.1` either.

**Comment by blink1073** [comment]
As in, you see the correct behavior in `0.12.1`?

**Comment by jakirkham** [comment]
Right.

**Comment by cgohlke** [comment]
Not really a bug IMO but certainly confusing. In this case, the image and meta-data of the first image series are sufficiently described by one page. Tifffile no longer reads superfluous TIFF pages if possible. But since there apparently is a use case, the previous behavior of the page sequence interface should probably be restored somehow. Will do.

**Comment by jakirkham** [comment]
This appears to be fixed with 0.14.0. Thanks @cgohlke and @blink1073. :)
